zip-0317.rst

```diff
@@ -241,6 +264,49 @@ described above) to the original conventional fee of 10000 zatoshis
 specified in [#zip-0313]_, and imposes a non-trivial cost for
 potential denial-of-service attacks.
 
+ZSA Fee calculation
+-------------------
+In addition to the parameters defined in the `Fee calculation`_ section, the ZSA protocol upgrade defines the following additional parameter:
+
+===================================== =============== ===============================================
+Parameter                                 Value         Units
+===================================== =============== ===============================================
+:math:`issuance\_fee`                 :math:`1000000` zatoshis per issuance action (as defined below)
+:math:`finalize\_fee`                 :math:`1000000` zatoshis per issuance action (as defined below)
+===================================== =============== ===============================================
+
+Wallets implementing this specification SHOULD use a conventional fee
+calculated in zatoshis per the following formula:
```

Comment:
Does the `zsa_conventional_fee` replace the `conventional_fee` after the ZSA upgrade? Or is it only used in certain circumstances?

This isn't clear, can we specify it here?

Reply:
Updated.

Reply:
Thanks!